pages/plant/[id].js

```diff
@@ -118,6 +139,13 @@ export default function PlantDetails({ plants }) {
           </StyledSeasonList>
         </IconContainer>
       </IconsContainer>
+      {/* Styling folgt nach Merge*/}
+      {isDeleteOption && (
+        <DeleteButton onClick={() => onDeletePlant(id)}>Delete</DeleteButton>
+      )}
+      <DeleteButton onClick={toggleDeleteOption}>
+        {toggleButtonName}
+      </DeleteButton>
     </>
   );
 }
```

Comment:
you could simplify this:
```suggestion
     {isDeleteOption ? (
  <DeleteButton onClick={() => onDeletePlant(id)}>Delete</DeleteButton>
) : (
  <DeleteButton onClick={toggleDeleteOption}>
    {toggleButtonName}
  </DeleteButton>
)}
```

Reply:
Thanks for the feedback. We use the second delete button in line 146 as cancel button. So we need to render it with the delete button in 144 if it isDeleteOption is true.

But we changed the styling name from the DeleteButtons, so ist clearer! :)